README.md

```diff
@@ -1,76 +1,40 @@
-Bitcoin Core integration/staging tree
+Tapyrus Core [![Build Status](https://github.com/chaintope/tapyrus-core/workflows/CI%20on%20Push/badge.svg?branch=master)](https://github.com/chaintope/tapyrus-core/actions?query=workflow%3A%22CI+on+Push%22)
 =====================================
 
-[![Build Status](https://travis-ci.org/bitcoin/bitcoin.svg?branch=master)](https://travis-ci.org/bitcoin/bitcoin)
+![tapyrus](doc/images/tapyrus-logo.png)
 
-https://bitcoincore.org
-
-What is Bitcoin?
+What is Tapyrus?
 ----------------
 
-Bitcoin is an experimental digital currency that enables instant payments to
-anyone, anywhere in the world. Bitcoin uses peer-to-peer technology to operate
-with no central authority: managing transactions and issuing money are carried
-out collectively by the network. Bitcoin Core is the name of open source
-software which enables the use of this currency.
+Tapyrus is a blockchain that has been forked from Bitcoin Core, and solves the governance issues.
+Bitcoin uses Proof of Work as a consensus algorithm, but Tapyrus creates a block (ie, approves a transaction)
+with multiple signatures from the Signer group which consists of industry stakeholders.
+Therefore, transactions on the blockchain are stably approved, and finality is given at the approved timing.
+Anyone can join the Tapyrus network, and anyone can access the public ledger and freely create and broadcast transactions.
```

Comment:
and reorder the last part as "and create and broadcast transactions freely."

Reply:
thx, I fix it.